lib/json0.js

```diff
@@ -663,6 +663,61 @@ json.transformComponent = function(dest, c, otherC, type) {
   return dest;
 };
 
+json.transformPresence = function(presence, op, isOwnOp) {
+  if (!presence || !isArray(presence.p)) return null;
+  if (!op) return presence;
+
+  presence = clone(presence);
+  op = clone(op);
+
+  // Deletions should be treated as insertions: transforming
+  // by them should result in a no-op (null presence), so
+  // let's just pretend that deletions are oi to get the
```

Comment:
The "pretend that deletions are oi" comment is confusing on first read, would be worth elaborating a bit more on how that works

Reply:
👍🏼  I've tried to clarify